src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt

```diff
@@ -99,7 +96,7 @@ class ProtectLevelCalculator(
 
   private fun getBreachRecallComplexityPoints(crn: String): Int {
     return when {
-      communityApiDataService.hasBreachedConvictions(crn) -> 1
+      communityApiDataService.hasBreachedConvictions(crn) -> 2
```

Comment:
It feels like there's a general idea that everything needs to be multiplied by 2, but that it's being done in quite a few different places